recipes/slow-buffer-to-buffer-alloc-unsafe-slow/src/workflow.ts

```diff
@@ -0,0 +1,291 @@
+import type { Edit, SgRoot, SgNode, Kinds } from "@codemod.com/jssg-types/main";
+import type Js from "@codemod.com/jssg-types/langs/javascript";
+import { getNodeImportStatements } from "@nodejs/codemod-utils/ast-grep/import-statement";
+import { getNodeRequireCalls } from "@nodejs/codemod-utils/ast-grep/require-call";
+
+export default function transform(root: SgRoot): string | null {
+	const rootNode = root.root();
+	const edits: Edit[] = [];
+	let hasChanges = false;
+
+	// Process require statements for 'buffer' module
+	hasChanges = processRequireStatements(root, edits) || hasChanges;
+
+	// Process ESM import statements for 'buffer' module
+	hasChanges = processImportStatements(root, edits) || hasChanges;
+
+	// Process SlowBuffer usage patterns (constructor calls, direct calls)
+	hasChanges = processSlowBufferUsage(rootNode, edits) || hasChanges;
+
+	// Process comments mentioning SlowBuffer
+	hasChanges = processSlowBufferComments(rootNode, edits) || hasChanges;
+
+	return hasChanges ? rootNode.commitEdits(edits) : null;
```

Comment:
```suggestion

	if (!edits.lenght) return null;

	return rootNode.commitEdits(edits);
```

No need to reduce code line. codemod need to be verbose to keep them maintainable.